spec/values/calculation/atan2.hrx

```diff
@@ -0,0 +1,204 @@
+
+<===> operation/simplification/output.css
+a {
+  b: atan2(2px + var(--c), 14px);
+}
+
+<===>
+================================================================================
+<===> preserved/y_arg/percentage/input.scss
+a {b: atan2(1.5px + 5%, 4px)}
+
+<===> preserved/y_arg/percentage/output.css
+a {
+  b: atan2(1.5px + 5%, 4px);
+}
+
+<===>
+================================================================================
+<===> preserved/x_arg/percentage/input.scss
+a {b: atan2(4px, 1.5px + 5%)}
+
+<===> preserved/x_arg/percentage/output.css
+a {
+  b: atan2(4px, 1.5px + 5%);
+}
+
+<===>
+================================================================================
+<===> preserved/variable/input.scss
+a {
+  b: atan2(var(--c));
+}
+
+<===> preserved/variable/output.css
+a {
+  b: atan2(var(--c));
+}
+
+<===>
+================================================================================
+<===> preserved/unknown_units/input.scss
+a {
+  b: atan2(1foo, 2bar);
+}
+
+<===> preserved/unknown_units/output.css
+a {
+  b: atan2(1foo, 2bar);
+}
+
+<===>
+================================================================================
+<===> error/incompatible_units/input.scss
+a {b: atan2(1, 1px)}
+
+<===> error/incompatible_units/error
+Error: 1 and 1px are incompatible.
+  ,
+1 | a {b: atan2(1, 1px)}
+  |             ^ 1
+  |                === 1px
+  '
+  input.scss 1:13  root stylesheet
+
+<===>
+================================================================================
+<===> error/one_arg/input.scss
+a {b: atan2(0)}
+
+<===> error/one_arg/error
+Error: 2 arguments required, but only 1 was passed.
+  ,
+1 | a {b: atan2(0)}
+  |       ^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/invalid_arg/input.scss
+a {b: atan2($)}
+
+<===> error/syntax/invalid_arg/error
+Error: Expected identifier.
+  ,
+1 | a {b: atan2($)}
+  |              ^
+  '
+  input.scss 1:14  root stylesheet
+
+<===>
+================================================================================
+<===> error/x_type/input.scss
+a {b: atan2(0, "0")}
+
+<===> error/x_type/error
+Error: Expected number, variable, function, or calculation.
+  ,
+1 | a {b: atan2(0, "0")}
+  |                ^
+  '
+  input.scss 1:16  root stylesheet
+
+<===>
+================================================================================
+<===> error/y_type/input.scss
+a {b: atan2("0", 0)}
+
+<===> error/y_type/error
+Error: Expected number, variable, function, or calculation.
+  ,
+1 | a {b: atan2("0", 0)}
+  |             ^
+  '
+  input.scss 1:13  root stylesheet
+
+<===>
+================================================================================
+<===> error/zero_args/input.scss
+a {b: atan2()}
+
+<===> error/zero_args/error
+Error: Expected number, variable, function, or calculation.
+  ,
+1 | a {b: atan2()}
+  |             ^
+  '
+  input.scss 1:13  root stylesheet
```

Comment:
You don't need to test _all_ argument numbers below the minimum, just verify that passing one too few args fails.